### PR TITLE
chore(charts): upgrade carbon charts to latest version and needs new color options

### PR DESCRIPTION
**Pull request by scottdickerson**
Closes #

**Summary**

- I updated @carbon/charts to the latest version and found that I now need to pass the carbon colors in the scale myself if they're not set by the user

**Change List (commits, features, bugs, etc)**

- fix(barChartUtils, TimeSeriesCard): update the formatColors payload to pass explicit colors in the scale options

**Acceptance Test (how to verify the PR)**

- Small regression test of the BarChartCard and TimeSeriesCard stories

### CODE REVIEW

File: src/constants/CardPropTypes.js [hunk 1]
```diff
@@ -14,6 +30,23 @@ import {
...
+  purple70,
+  cyan50,
+  teal70,
+  magenta70,
+  red50,
+  red90,
+  green60,
+  blue80,
+  magenta50,
+  // yellow50,
```
Comment (marcJV) [inline src/constants/CardPropTypes.js:43]: we should remove the commented colors, or uncomment them

**Review by marcJV — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -126,9 +126,9 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.10.2",
-    "@carbon/charts": "^0.36.2",
+    "@carbon/charts": "^0.40.8",
     "@carbon/charts-react": "^0.36.2",
-    "@carbon/colors": "10.8.0",
+    "@carbon/colors": "^10.15.0",
```
Comment (tay1orjones) [inline package.json:131]: This should be pinned, not caret range.
```suggestion
    "@carbon/colors": "10.15.0",
```

File: src/constants/CardPropTypes.js [hunk 3]
```diff
@@ -1,4 +1,20 @@
...
+  magenta70,
+  purple70,
+  red50,
+  red90,
+  teal50,
+  teal70,
+  cyan90,
+  purple50,
+  // orange70,
+  // yellow50,
```
Comment (tay1orjones) [inline src/constants/CardPropTypes.js:16]:
```suggestion
```

File: src/constants/CardPropTypes.js (hunk 1 above)
Comment (tay1orjones) [inline src/constants/CardPropTypes.js:43]:
```suggestion
```

File: src/constants/CardPropTypes.js [hunk 4]
```diff
@@ -14,6 +30,23 @@ import {
...
+  magenta70,
+  red50,
+  red90,
+  green60,
+  blue80,
+  magenta50,
+  // yellow50,
+  teal50,
+  cyan90,
+  // orange70,
```
Comment (tay1orjones) [inline src/constants/CardPropTypes.js:46]:
```suggestion
```

**Review by tay1orjones — CHANGES_REQUESTED** [review]

**Comment by scottdickerson** [comment]
@tay1orjones @marcJV I fixed the issues and the testcases, can you please take another look?

**Review by tay1orjones — APPROVED** [review]

**Review by JoelArmendariz — APPROVED** [review]
LGTM

**Pull request merged**